File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/RaspunsuriBanciReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class RaspunsuriBanciReportFilterModel {
	
	private String denumireBanca;
	private Long proiectId; 
	private Date termenRaspunsDela;
	private Date termenRaspunsPanaLa;

	public String getDenumireBanca() {
		return denumireBanca;
	}
	public void setDenumireBanca(String denumireBanca) {
		this.denumireBanca = denumireBanca;
	}
	public Long getProiectId() {
		return proiectId;
	}
	public void setProiectId(Long proiectId) {
		this.proiectId = proiectId;
	}
	public Date getTermenRaspunsDela() {
		return termenRaspunsDela;
	}
	public void setTermenRaspunsDela(Date termenRaspunsDela) {
		this.termenRaspunsDela = termenRaspunsDela;
	}
	public Date getTermenRaspunsPanaLa() {
		return termenRaspunsPanaLa;
	}
	public void setTermenRaspunsPanaLa(Date termenRaspunsPanaLa) {
		this.termenRaspunsPanaLa = termenRaspunsPanaLa;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/raspunsuri-banci-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "@app/shared/json-mapper";

@JsonObject
export class RaspunsuriBanciReportFilterModel {

    @JsonProperty("denumireBanca", String)
    public denumireBanca: string = null;

    @JsonProperty("proiectId", Number)
    public proiectId: number = null;

    
    @JsonProperty("termenRaspunsDela", JsonDateConverter)
    public termenRaspunsDela: Date = null;
    
    @JsonProperty("termenRaspunsPanaLa", JsonDateConverter)
    public termenRaspunsPanaLa: Date = null;
}